.github/workflows/review.yaml

```
@@ -0,0 +1,18 @@
+name: 👀 Review
+on:
+  push:
+    branches:
+      - "**"
+      - "!master"
+      - "!main"
+      - "!feat/**"
+      - "!fix/**"
```

```suggestion
on:
  pull_request:
    branches:
      - "!master"
      - "!main"
    tags-ignore:
      - v*
```

Je pense pas besoin de faire de branches si on est juste en `push`, ça pourrait faire économiser du cpu

---

Je crois qu'il ne fait pas de branche ici car elle n'est pas automatique. D'ailleurs dans ce cas, comment on l'a déclenche si besoin et qu'on ne respecte pas la syntaxe fix/feat ?

---

ce sera comme pour la mise en prod manuelle sur certains produits, en fait il y aura un pipeline en attente de validation